client.cpp

```diff
@@ -878,6 +878,130 @@ bool verify_client::finished(void)
 
 ///////////////////////////////////////////////////////////////////////////
 
+crc_verify_client::verify_request::verify_request(request_type type,
+                                              unsigned int size,
+                                              struct timeval* sent_time,
+                                              unsigned int keys,
+                                              const char *key,
+                                              unsigned int key_len) :
+        client::request(type, size, sent_time, keys),
+        m_key(NULL), m_key_len(0)
+{
+    m_key_len = key_len;
+    m_key = (char *)malloc(key_len);
```

Comment:
why not using new/delete

Reply:
I changed it to new/delete[]. Using autopointer is a bad idea. for one, it calls delete and not delete[] for arrays which can result in an undefined behavior. Also, it was deprecated from C11, there's a reason.